Dockerfile

```diff
@@ -199,7 +176,6 @@ FROM debian-base AS debian
 
 # Add atlantis user to Debian as well
 RUN useradd --create-home --user-group --shell /bin/bash atlantis && \
-    adduser atlantis root && \
```

Comment:
same with this, the user needs to be added

Reply:
The user is added on the previous line. The line deleted is adding atlantis user to the root group I believe.

Reply:
Yep that's right, `adduser atlantis root` `adduser` creates a new user, but we already do that in the line above (useradd) and `atlantis root` maps the atlantis user to a root group.